ansible/roles/kong-api/defaults/main.yml

```diff
@@ -5543,9 +5543,6 @@ kong_apis:
       - name: jwt
```

Comment:
if no authorization is required remove the jwt.

Reply:
done